gradle/libs.versions.toml

```diff
@@ -78,6 +78,7 @@ mockserver = "5.15.0"
 nessie = "0.106.0"
 netty-buffer = "4.2.9.Final"
 object-client-bundle = "3.3.2"
+opentelemetry-httpclient = "2.20.1-alpha"
```

Comment:
please check on the licence part, if we are ok and if we need to update it.

Reply:
Could you let me know how to check? https://github.com/apache/iceberg/blob/main/aws-bundle/LICENSE you mentioned before only contains dependencies used in `aws-bundle` module. `iceberg-core` module doesn't have LICENSE file.

Reply:
Ah i see, what i wanted to mostly know is.the following : 
1. can we take a dependency on this, based on some ASF doc search it seems like we very well can :) 
2. LICENCE : Apologies i am not an expert in this, just wanted to be cautious on this before we introduce a new dependency to project and update the LICENCE if required accordingly.

Lets do this : once the CI is green (i believe https://github.com/apache/iceberg/pull/14360/changes/a70abe35ca49434d2d6bec2a4defd0d2e106d500 addresses my latest feedbacks :)) i will go ahead and approve ! (thanks a lot for change, i can truly see the benefits of this) and lets wait for sometime for some licence | dependency experts to weigh in (i will ping some folks on slack too) before we go ahead and merge, WDYT ?